code/components/citizen-server-impl/src/state/ServerGameState.cpp

```diff
@@ -3244,6 +3244,24 @@ struct CVehicleComponentControlEvent
 	MSGPACK_DEFINE_MAP(vehicleGlobalId, pedGlobalId, componentIndex, request, componentIsSeat, pedInSeat);
 };
 
+struct CClearPedTasksEvent
+{
+	void Parse(rl::MessageBuffer& buffer)
+	{
+		pedId = buffer.Read<uint16_t>(13);
+		unk1 = buffer.Read<uint8_t>(1);
+	}
+
+	inline std::string GetName()
+	{
+		return "clearPedTasksEvent";
+	}
+
+	int pedId;
+	bool unk1;
```

Comment:
Should double check this in your testing, but I believe unk1 corresponds to: ``Clear Tasks Immediately``

Reply:
It does appear so, I'll change the name of it then.